### PR TITLE
Rafactor of opPool

### LINKED ISSUE #295: [OpPool] unstub removal methods

### PR DESCRIPTION
**Pull request by mpetrunic**
- added missing rremoval methods
- disabled attestation removal, it should be handled depending on #307
- split opPool into tematic modules

We could split database into similar modules (#308). That should enable us to abstract most of this modules. 

resolves #295

### CODE REVIEW

**Review by GregTheGreek — DISMISSED** [review]
LGTM
Only thing would be to perhaps rename `all` to `getAll`

File: src/db/schema.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export enum Bucket {
   mainChain, // slot -> blockHash
   chainInfo, // Key -> number64 | stateHash | blockHash
   validator,
-  genesisDeposit, // index -> Deposit
+  deposit, // index -> Deposit
```
Comment (wemeetagain) [inline src/db/schema.ts:16]: :+1:

File: src/opPool/modules/index.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export * from "./attestation";
```
Comment (wemeetagain) [inline src/opPool/modules/index.ts:1]: Can you export all of the opPool `OperationsModule`s here and use this as the import file in `src/opPool//opPool.ts`?

**Review by wemeetagain — CHANGES_REQUESTED** [review]
I like this a lot, feels very clean.

Only one minor comment.

File: src/opPool/modules/index.ts (hunk 2 above)
Comment (mpetrunic) [reply src/opPool/modules/index.ts:1]: oh lol, I meant to do this, but it seems I forgot to o rest of it haha

**Review by wemeetagain — APPROVED** [review]
LGTM!

**Review by GregTheGreek — APPROVED** [review]

**Pull request merged**